src/amber/pipes/cors.cr

```diff
@@ -132,8 +132,8 @@ module Amber
         end
       end
 
-      def any?
-        @origins.includes? "*"
+      def empty?
```

Comment:
I think `any` is a better name here.  The `*` symbol represents "any origin" as in the wildcard symbol.  Empty is a common method on a string and may confuse someone into thinking it is checking for an empty string.

Reply:
That's a good point, I hadn't given it enough thought since I was just going through and fixing what Ameba said was wrong. I'll probably change from `#any?` to `#any_origin?` to stop Ameba from complaining and make it clearer what we are checking for with this method.